invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/RecordsSearch/RecordsSearch.js

```diff
@@ -0,0 +1,232 @@
+import React, { Component } from 'react';
+import { generatePath } from 'react-router';
+import {
+  Container,
+  Grid,
+  Segment,
+  Icon,
+  Header,
+  Responsive,
+  Accordion,
+  Dropdown,
+  Menu,
+} from 'semantic-ui-react';
+import {
+  ReactSearchKit,
+  SearchBar,
+  ResultsList,
+  ResultsLoader,
+  ResultsPerPage,
+  EmptyResults,
+  Error,
+  Pagination,
+  Count,
+  SortBy,
+  SortOrder,
+  Aggregator,
+} from 'react-searchkit';
+import { apiConfig } from '../../../common/api/base';
+import { FrontSiteURLS } from '../../../common/urls';
+import { Error as IlsError } from '../../../common/components';
+import { document as documentApi } from '../../../common/api';
+import { SearchBar as RecordsSearchBar } from '../../../common/components';
+import { ResultsList as RecordsResultsList } from './components';
+import { default as config } from './config';
+import './RecordsSearch.scss';
+
+export class RecordsSearch extends Component {
+  state = { activeIndex: 0 };
+
+  handleClick = (e, titleProps) => {
```

Comment:
Perhaps we could use a more descriptive name here that describes what kind of click callback it is.